build.sbt

```diff
@@ -67,9 +69,16 @@ lazy val frontEnd = project("front-end")
       "org.webjars" % "react" % "0.14.3",
       "org.webjars" % "react-router" % "1.0.3",
       "org.webjars" % "jquery" % "2.2.0",
-      "org.webjars" % "foundation" % "5.3.0"
+      "org.webjars" % "foundation" % "5.3.0",
+      "com.typesafe.conductr" %% "lagom10-conductr-bundle-lib" % "1.4.1"
```

Comment:
This isn't needed since we use sbt-lagom-bundle 1.0.2 (because https://github.com/typesafehub/sbt-lagom-bundle/blob/master/src/main/scala/com/typesafe/sbt/bundle/LagomBundle.scala#L44)